worker/src/core/loaders/message-worker/index.ts

```diff
@@ -169,6 +170,45 @@ const finalize = tracer.wrap(
     const logGovsgJob = postmanConnection
       .query('SELECT log_next_job_govsg();')
       .then(([result]) => get(result, '[0].log_next_job_govsg', ''))
+      .then(async (campaignId: number) => {
+        const rawMessages = await postmanConnection.query(
+          "SELECT * FROM govsg_messages WHERE status = 'ERROR' AND campaign_id = :campaignId",
+          { type: QueryTypes.SELECT, replacements: { campaignId } }
+        )
+        if (!rawMessages) {
+          return campaignId
+        }
+
+        const messagesWithErr = rawMessages as Array<{
+          id: number
+          error_code: string
+          error_description: string
+        }>
+        if (
+          messagesWithErr.length === 0 ||
+          !config.get('sgcCampaignAlertChannelWebhookUrl')
```

Comment:
Not blocking but:

Should we be throwing when `!config.get('sgcCampaignAlertChannelWebhookUrl')`? We want to be alerted, right?